python/cudf_polars/cudf_polars/callback.py

```diff
@@ -174,7 +175,9 @@ def execute_with_cudf(
     device = config.device
     memory_resource = config.memory_resource
     raise_on_fail = config.config.get("raise_on_fail", False)
-    if unsupported := (config.config.keys() - {"raise_on_fail"}):
+    parquet_options = config.config.get("parquet_options", {})
+    cudf_polars.dsl.translate.ir.parquet_options = parquet_options
```

Comment:
This was just a quick hack to get things working, but we should decide on a way we're happy with for configuring the reader as well as possibly other configurable bits. 

High level questions:
- Should we have a global config, or a per query config (per GPU-Engine) 
- If per query, how should we pass it around? Should `translate_ir` take another argument and make it available to each overload of `_translate_ir`? 


cc @wence-